frontend/src/metabase/querying/components/FilterModal/CoordinateFilterEditor/CoordinateFilterEditor.tsx

```diff
@@ -85,23 +84,26 @@ export function CoordinateFilterEditor({
 }
 
 interface NumberValueInputProps {
-  column: Lib.ColumnMetadata;
   values: NumberValue[];
   valueCount: number;
   hasMultipleValues?: boolean;
   onChange: (values: NumberValue[]) => void;
 }
 
 function NumberValueInput({
-  column,
   values,
   valueCount,
   hasMultipleValues,
   onChange,
 }: NumberValueInputProps) {
   if (hasMultipleValues) {
     return (
-      <FilterValuePicker value={values} column={column} onChange={onChange} />
+      <FilterValuePicker
+        values={values.map(value => String(value))}
+        placeholder={t`Enter a number`}
+        getCreateLabel={query => (isFinite(parseFloat(query)) ? query : null)}
+        onChange={values => onChange(values.map(value => Number(value)))}
```

Comment:
```suggestion
        onChange={values => onChange(values.map(value => parseFloat(value)).filter(value => isFinite(value)))}
```

Same comment for 3 other places

Reply:
I'll address that in the next PR where the component is changed